internal/workflow/activities/storage.go

```diff
@@ -2,15 +2,86 @@ package activities
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"time"
 
 	"github.com/google/uuid"
 	"github.com/oklog/run"
+	"go.artefactual.dev/tools/temporal"
 	temporalsdk_activity "go.temporal.io/sdk/activity"
+	goa "goa.design/goa/v3/pkg"
 
 	goastorage "github.com/artefactual-sdps/enduro/internal/api/gen/storage"
+	"github.com/artefactual-sdps/enduro/internal/storage"
 )
 
+type CreateStoragePackageActivity struct {
+	client storage.Client
+}
+
+type CreateStoragePackageActivityParams struct {
+	Name       string
+	AIPID      string
+	ObjectKey  string
+	Status     string
+	LocationID string
```

Comment:
Should we reuse the `goastorage.CreatePayload` struct and do the transformation from strings outside?

Reply:
It's probably better as it is right now, ignore me.